src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -973,7 +974,7 @@ public void combineAllStudentCommits(Repository repository, ProgrammingExercise
             var optionalStudent = ((StudentParticipation) repository.getParticipation()).getStudents().stream().findFirst();
             var name = optionalStudent.map(User::getName).orElse(artemisGitName);
             var email = optionalStudent.map(User::getEmail).orElse(artemisGitEmail);
-            studentGit.commit().setMessage("All student changes in one commit").setCommitter(name, email).call();
+            studentGit.commit().setSign(false).setMessage("All student changes in one commit").setCommitter(name, email).call();
```

Comment:
I suggest to refactor this into a method so that we do not forget to set this in future usages.
The method could take the git object, and create a CommitCommand in which `setSign(false)` is invoked

Reply:
thank you for the feedback, I added a method to both GitService and GitUtilService